docs/getting-started.rst

```diff
@@ -81,6 +81,28 @@ directories:
 * ``verify.yml`` is the Ansible file used for testing as Ansible is the default :ref:`verifier`. This
   allows you to write specific tests against the state of the container after
   your role has finished executing. Other verifier tools are available (Note that :std:doc:`TestInfra <testinfra:index>` was the default verifier prior to molecule version 3).
+  
+.. note::
+
+    If the `verify.yml` playbook does not explicitly `include_role` your role, the `library` and `module_utils`
```

Comment:
RST required double backquotes, single ones render as italic instead. See https://molecule--3222.org.readthedocs.build/en/3222/getting-started.html

Reply:
thanks - yeah, I tried it with the "Edit on Github" feature but in the end writing clean RST that way isn't all too easy...

I now installed the pre-commit hooks locally and built the docs; the issues should be fixed.